docs/index.md

```diff
@@ -4,18 +4,19 @@
 documentation and implementation of the Mozilla telemetry ingestion system
 deployed to Google Cloud Platform (GCP).
 
-There are currently two components:
+The components are:
 
 - [ingestion-edge](./ingestion-edge/index.md): a simple Python service for accepting HTTP
   messages and delivering to Google Cloud Pub/Sub
 - [ingestion-beam](ingestion-beam): a Java module defining
   [Apache Beam](https://beam.apache.org/) jobs for streaming and batch
   transformations of ingested messages
+- ingestion-sink (documentation pending): a Java application that runs
+  in Kubernetes, reading input from Google Cloud Pub/Sub and emitting
+  records to batch-oriented outputs like GCS or BigQuery
 
 The design behind the system along with various trade offs are documented in
-the architecture section. Note that as of this writing (August 2019) this
-GCP ingestion is changing quickly, so some parts of this documentation may be out
-of date.
+the architecture section.
```

Comment:
Do we want to link to the architecture section here?

```suggestion
the [architecture section](./architecture/overview.md).
```

Reply:
I missed this suggestion before merging 😢 Will follow up